### ISSUE TITLE
test result dice : 0.000

### ISSUE THREAD

**Issue by scwuchung** [askerlee/segtran#37]
thanks for sharing this great project , and i met some problems when use it for LUNA16  dataset segmentation .the training loss is low ,but the test reaults are :dice: 0.000, jc: 0.000, hd: nan, asd: nan .can you give me some advice ?thanks.

**Comment by askerlee** [comment]
Seems the groundtruth labels are not correctly loaded?

**Comment by scwuchung** [comment]
> Seems the groundtruth labels are not correctly loaded?

thanks for reply . i will check this.
